Code/Engine/Core/World/Component.h

```diff
@@ -92,14 +92,11 @@ class EZ_CORE_DLL ezComponent : public ezReflectedClass
 
 
   /// \brief Sends a message to this component.
-  bool SendMessage(ezMessage& msg);
-  bool SendMessage(ezMessage& msg) const;
-
-  /// \brief Queues the message for the given phase and processes it later in that phase.
-  void PostMessage(const ezMessage& msg, ezObjectMsgQueueType::Enum queueType) const;
+  EZ_ALWAYS_INLINE bool SendMessage(ezMessage& msg) { return SendMessageInternal(msg, false); }
```

Comment:
I don't really understand why this extra bool is needed?

Reply:
SendMessage always returned the bool. PostMessage does not.

Reply:
It just shows them in the diff side-by-side, because one of the PostMessage variants was removed (was redundant).

Reply:
I mean this bool bWasPostedMsg thing. Why is that needed?